addon/custom-map-with-default.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+class CustomMapWithDefault extends Ember.MapWithDefault {
```

Comment:
@stefanpenner I'm extending MapWithDefault this way now instead of just highjacking the prototype directly